src/commons/layouts/base/Container/Container.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
+    display: inline-flex;
+    position: absolute;
```

Comment:
And do you really need pos: a here? I think the container should work without it. This style might broke something in the future.